.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish Fireshape's Docker images
+
+on:
+  # Push to master or PR
+  push:
+    branches:
+      - master
+  pull_request:
+  # build every week to stay up-to-date with firedrake
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
change this to: `cron 0 0 * * 1` and you will get the Firedrake image built the night before rather than the week before :slightly_smiling_face:

Reply:
Great!